roboteam_ai/src/treeinterp/Switches.cpp

```diff
@@ -234,8 +241,8 @@ bt::Node::Ptr Switches::tacticSwitch(std::string name, bt::Blackboard::Ptr prope
             }
             },
             {"PassTactic", {
-                    {"passOne", robotType::random},
-                    {"passB", robotType::random}
+                    {"passer", robotType::random},
```

Comment:
I could remove this, the pass tactic will realistically not be used anyways since passing will be incorporated in other tactics anyways. For now I will replace passer by closeToBall